package.json

```diff
@@ -46,5 +46,13 @@
   },
   "lint-staged": {
     "*.{js,css,md,ts,tsx,json,yaml,yml}": "prettier --write"
+  },
+  "resolutions": {
+    "@walletconnect/client": "2.0.0-beta.20.3",
+    "@walletconnect/types": "2.0.0-beta.20.3",
+    "@walletconnect/utils": "2.0.0-beta.20.3",
+    "@walletconnect/client-v1": "npm:@walletconnect/client@1.6.6",
+    "@walletconnect/types-v1": "npm:@walletconnect/types@1.6.6",
+    "@walletconnect/utils-v1": "npm:@walletconnect/utils@1.6.6"
```

Comment:
Why are resolutions needed in the base package here in addition to the ones in the subpackages?